### PR TITLE
Drafting Privacy and Security Considerations

### PR DESCRIPTION
**Pull request by paulbastian**
gathering some ideas on privacy and security considerations

### CODE REVIEW

File: draft-looker-jwt-cwt-status-list.md [hunk 1]
```diff
@@ -154,11 +154,35 @@ TODO add more
...
-TODO elaborate on heard privacy
+## Herd Privacy 
+TODO elaborate on herd privacy, size of the status list
+
+## Profiling
+TODO elaborate on Verifiers regularly fetching the status list to create a profile, possible countermeasures with authorized access to the status list
+
+## Correlation Risks and Tracking
+TODO elaborate on Issuer-Verifier correlation and Verifier-Verifier correlation as the status list introduces unique,trackable data
+TODO elaborate on issuers avoiding sequential usage of indices and status lists
```
Comment (c2bo) [inline draft-looker-jwt-cwt-status-list.md:179]:
```suggestion
TODO elaborate on issuers avoiding sequential usage of indices and status lists
```

File: draft-looker-jwt-cwt-status-list.md [hunk 2]
```diff
@@ -154,11 +154,35 @@ TODO add more
...
+## Authorized access to the Status List
+TODO elaborate on authorization mechanisms preventing misuse and profiling as described in privacy section
+
+## History
+TODO elaborate on status list only providing the up-to date/latest status, no historical data, may be provided by the underlying hosting architecture
 
 # Privacy Considerations
 
-TODO elaborate on heard privacy
+## Herd Privacy
```
Comment (c2bo) [inline draft-looker-jwt-cwt-status-list.md:171]:
```suggestion
## Herd Privacy
```

**Review by c2bo — COMMENTED** [review]
Need to remove the trailing whitespaces for the markdown tooling to be happy

**Comment by tplooker** [comment]
Looking good @paulbastian you should be able to run 'make fix-lint' locally to fix the whitespace issues

**Pull request merged**